dev_guide/builds.adoc

```diff
@@ -2529,15 +2532,42 @@ list of custom labels that will be applied to each image built from the BuildCon
 ====
 [source,yaml]
 ----
-output:
-  to:
-    kind: "ImageStreamTag"
-    name: "my-image:latest"
-  imageLabels:
-  - name: "vendor"
-    value: "MyCompany"
-  - name: "authoritative-source-url"
-    value: "registry.mycompany.com"
+spec:
+  output:
+    to:
+      kind: "ImageStreamTag"
+      name: "my-image:latest"
+    imageLabels:
+    - name: "vendor"
+      value: "MyCompany"
+    - name: "authoritative-source-url"
+      value: "registry.mycompany.com"
+----
+====
+
+[[output-image-digest]]
+=== Output Image Digest
+
+Built images can be uniquely identified by their
+link:++https://docs.docker.com/registry/spec/api/#/content-digests++[digest], which can
```

Comment:
@ahardin-rh I'm not sure of another workaround. I think only the Customer Portal build would potentially have an issue; worth checking with Lee on. @vikram-redhat Can you look into that?

I'd be inclined to merge this but mark for follow-up before it gets downstream.